common/config/rush/pnpm-lock.yaml

```diff
@@ -9,7 +9,7 @@ importers:
     specifiers:
       '@playwright/test': ^1.24.1
     devDependencies:
-      '@playwright/test': 1.31.2
```

Comment:
Seems like dependencies other than `sinon` were updated as well, did you by any chance run `rush update --full`? Was that intentional?

Reply:
i doubt doing the full update was intentional; but it might be a good thing; now that you guys have broken off from the core repo, how often are you guys doing full updates? core used to do it whenever there was a new addon

Reply:
It wasn’t intentional but yes I did run full rush update. Let me know if I should revert. Thanks